e2e/vmss.go

```diff
@@ -66,19 +64,14 @@ func createVMSSWithPayload(ctx context.Context, customData, cseCmd, vmssName str
 	}
 
 	isAzureCNI, err := opts.clusterConfig.IsAzureCNI()
-	if err != nil {
-		return nil, fmt.Errorf("determine whether chosen cluster uses Azure CNI from cluster model: %w", err)
-	}
+	require.NoError(t, err)
 
 	if isAzureCNI {
-		if err := addPodIPConfigsForAzureCNI(ctx, &model, vmssName, opts); err != nil {
-			return nil, fmt.Errorf("create pod IP configs for azure CNI scenario: %w", err)
-		}
+		err = addPodIPConfigsForAzureCNI(ctx, &model, vmssName, opts)
+		require.NoError(t, err)
```

Comment:
what kind of output do we get  when this fails ?  I'm hoping the line number and/or function/ test name are all reported ?

Reply:
any reason why we don't add add more context to the assertion using `msgAndArgs`

Reply:
Something similar to this:
```
    scenario_helpers_test.go:129: 
        	Error Trace:	/Users/r2k1/p/AgentBaker/e2e/scenario_helpers_test.go:129
        	            				/Users/r2k1/p/AgentBaker/e2e/scenario_helpers_test.go:45
        	            				/Users/r2k1/p/AgentBaker/e2e/scenario_test.go:369
        	Error:      	Received unexpected error:
        	            	error waiting for vmss abtest-2024-07-13-y6cr-marinerv2ChronyRestarts ready: failed to find or wait for node to be ready: context deadline exceeded
```

Reply:
Add more context back.

In general, with log messages grouping by test it becomes easier to decipher log messages